jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/exec/EcsExecUtils.kt

```diff
@@ -95,6 +105,78 @@ object EcsExecUtils : CoroutineScope by ApplicationThreadPoolScope("EcsExec") {
         )
     }
 
+    fun getEc2InstanceTaskRoleArn(project: Project, clusterArn: String, ecsClient: EcsClient, task: software.amazon.awssdk.services.ecs.model.Task): String? {
```

Comment:
probably do something like:
```
import software.amazon.awssdk.services.ecs.model.Task as EcsTask
```
instead of using fully qualified class name

Reply:
Will change this